Docs for 1.13 release (#7515)
solo-io/gloo: docs/content/introduction/faq.md

kind: List
{{< /highlight >}}

> In this example, you can see the Gateway and VirtualService objects are merged into the Proxy object and this is the object that will drive the Envoy xDS/configuration model. To see *exactly* what the Envoy configuration is:

Reviewer: ```suggestion
In this example, you can see the Gateway and VirtualService objects are merged into the proxy that then drives the Envoy xDS/configuration model. To see *exactly* what the Envoy configuration is:
```
